src/components/users/TickCard.tsx

```diff
@@ -37,7 +38,10 @@ export default function TickCard ({ tickId, ticks, setTicks, dateClimbed, notes,
         <p className='text-sm text-gray-500 mb-0'>{notes}</p>
       </div>
       <div className='flex flex-row'>
-        <p className='text-sm text-gray-500 pr-5'>{style}</p>
+        <div className='flex flex-col'>
```

Comment:
This shows the attemptType on the tick card: 
![Screenshot 2025-01-25 at 6 30 44 PM](https://github.com/user-attachments/assets/63de3e51-d95b-4792-a89a-e4aeb5801567)